docs/tutorials/AwsBatch101.md

```diff
@@ -137,13 +106,52 @@ backend {
     }
   }
 }
+
 ```
 
-**Run Workflow**
+For more information about this configuration or how to change the behaviour of AWS Batch, visit the [AWS Backend](/backends/AWSBatch) page.
+
 
-`java -Dconfig.file=aws.conf -jar cromwell-36.jar run hello.wdl -i hello.inputs`
+#### Workflow Source Files 
 
-**Outputs**
+Lastly, create an example workflow to run. We're going to define a simple workflow that will `echo` a string to the console and return the result to Cromwell. Within AWS Batch (like other cloud providers), we're required to specify a Docker container for every task.
+
+##### `hello.wdl`
+
+```wdl
+task hello {
+  String addressee = "Cromwell"
+  command {
+    echo "Hello ${addressee}! Welcome to Cromwell . . . on AWS!"
+  }
+  output {
+    String message = read_string(stdout())
+  }
+  runtime {
+    docker: "ubuntu:latest"
+  }
+}
+
+workflow wf_hello {
+  call hello
+  output { hello.message }
+}
+```
+
+#### Running Cromwell and AWS 
+
+Provided all of the files are within the same directory, we can run our workflow with the following command:
+
+```bash
+java -Dconfig.file=aws.conf -jar cromwell-36.jar run hello.wdl
```

Comment:
Maybe add a:
> **Note**: You might have a different Cromwell version number here

Reply:
I like that (e753538096303d4f88e80cafe82ecc23aa11876c)!